addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1688,8 +1688,8 @@
 
 		"141"	//Frontier Justice
 		{
-			"desp"			"Frontier Justice: {positive}Crits when your sentry is targeting the boss"
-			
+			"desp"			"Frontier Justice: {positive}Crits when your sentry is targeting the boss, {negative} Increased spread, slower reload speed"
```

Comment:
Should make desp match with TF2 attributes:
`, {negative}15% less accurate, 50% slower reload time`

Reply:
resolved